src/Sleep.js

```diff
@@ -0,0 +1,26 @@
+class Sleep {
+  constructor() {}
+  userSleepData(sleepData, id) {
+    this.userSleep = sleepData.filter(dailySleep => dailySleep.userID === id);
+    return this.userSleep;
+  }
+  averageAllTimeSleep() {
+    let allTimeSleep = this.userSleep.reduce((sleep, day) =>{
+      sleep += day.sleepQuality;
+      return sleep 
+    }, 0)
+    return Number((allTimeSleep / this.userSleep.length).toFixed(1))
+  }
+  daySleep(dateSelected) {
```

Comment:
I think that we might be able to use the all time data in this section on a refactor